CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+
+* Deprecate `amount` property on `DropInRequest`
```

Comment:
The Android Drop-In release script updates the CHANGELOG file programmatically, so adding this now will cause a duplicate entry.

Reply:
Oh right, I forgot about that. I think we should either update the iOS release script to do the same or remove that part of the Android script so that we're consistent across repos.

Reply:
Voting to remove that portion of the Android release script after discussions in stand up.